Misc/NEWS.d/3.7.0a1.rst

```diff
@@ -5264,16 +5264,6 @@ Added missed names to PC/python3.def.
 
 ..
 
-.. bpo: 28762
-.. date: 0035
-.. nonce: Ru0YN_
-.. section: Build
-
-lockf() is available on Android API level 24, but the F_LOCK macro is not
-defined in android-ndk-r13.
```

Comment:
Since Python 3.7a1 has been released, I dislike the idea of modying the changelog. Please leave it as it is. Add a new NEWS entry if you want to document the change.